src/Fixer/Operator/NotOperatorWithSuccessorSpaceFixer.php

```diff
@@ -50,6 +44,26 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     /**
      * {@inheritdoc}
      */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Logical NOT operators (!) should have one trailing whitespace.',
```

Comment:
these are on the list to be merged into `binary_operator_spaces` with config, resolving even more conflicts